public/app/plugins/datasource/elasticsearch/datasource.ts

```diff
@@ -245,6 +238,25 @@ export class ElasticDatasource
     return this.request('POST', url, data, { 'Content-Type': 'application/x-ndjson' });
   }
 
+  private async isEs7Cached(): Promise<boolean> {
```

Comment:
I am slightly confused about naming here, because we are not doing any caching here and we are not checking for cached value, so why is it called `isEs7Cached()`?

Reply:
you are totally correct 😁 , that was meant to be called `isEs6Uncached`, signalizing that it will always go and do the checking.

Reply:
i adjusted this in the code.